web/app/src/main.js

```diff
@@ -3,6 +3,4 @@ import App from './App.vue'
 import './index.css'
 import router from './router'
 
-export const SERVER_URL = process.env.NODE_ENV === 'production' ? '' : 'http://localhost:8080'
-
```

Comment:
This would break the ability to run the backend on :8080 while running the front-end on another port
(`make run` followed by `make frontend-run`)

Reply:
That's right. I'll take a look how this could be resolved